src/test/java/io/visual_regression_tracker/sdk_java/PathProviderTest.java

```diff
@@ -0,0 +1,33 @@
+package io.visual_regression_tracker.sdk_java;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class PathProviderTest {
+
+  String apiUrl = "http://localhost:4200";
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/Visual-Regression-Tracker/sdk-java/pullRequest?prid=6309193)